screens/Home/index.tsx

```diff
@@ -70,12 +70,10 @@ export default function HomeScreen() {
     }, [isAuthenticated, refetchPoints]),
   )
   const { isPlaceholderData } = useMySignal()
-
   const handleWebsiteOpen = async () => {
-    await WebBrowser.openBrowserAsync(
-      "https://7axab-zyaaa-aaaao-qjv7a-cai.icp0.io/",
-    )
+    await WebBrowser.openBrowserAsync(process.env.POINTS_CANISTER_URL || "")
```

Comment:
This is bad because the url might sometimes fallback to an empty screen. 


https://github.com/user-attachments/assets/0a8e1dec-a9d3-465d-ace9-2c648abc0586


Instead you can add an if condition to check if we have env variable and await the webBrowser.
```suggestion
    if (canisterLink) {
      await WebBrowser.openBrowserAsync(canisterLink)
    }
```